bberg/src/relation_builder.rs

```diff
@@ -346,7 +346,7 @@ fn craft_expression<T: FieldElement>(
                 return (
                     1,
                     format!(
-                        "FF(uint256_t{{{}, {}, {}, {}}})",
+                        "FF(uint256_t{{{}LLU, {}LLU, {}LLU, {}LLU}})",
```

Comment:
UL should do the job as we need integers of 64 bits, i.e., unsigned long on this architecture.
I am however very puzzled on how it compiled in the past as we have occurences like 2^48.